hazelcast/src/main/java/com/hazelcast/logging/Logger.java

```diff
@@ -57,7 +57,7 @@ private Logger() {
     /**
      * Obtains a {@link ILogger logger} for the given {@code clazz}.
      *
-     * @param clazz the class to obtain the logger for.
+     * @param clazz the class of the logger to obtain.
```

Comment:
nit: I think the old javadoc was closer to the truth, as `clazz` is not `the class of the logger`, but the class for which the logger is created/obtained.